pom.xml

```diff
@@ -421,4 +421,4 @@
         </plugins>
     </build>
 
-</project>
\ No newline at end of file
+</project>
```

Comment:
This seems fishy.

Reply:
On the updated branch I didn't include this as it seems unnecessary.